workshops/views.py

```diff
@@ -937,7 +937,6 @@ def all_events(request):
 
 
 @login_required
-@permission_required('workshops.add_todoitem', raise_exception=True)
```

Comment:
This is good for now (but please mark with a comment why we check perms in the middle of the view). If we have more forms in that view in future we'll change that conditional.